Source/SourceKittenFramework/String+SourceKitten.swift

```diff
@@ -301,9 +310,10 @@ extension String {
     /// Returns a copy of `self` with the leading whitespace common in each line removed.
     public func stringByRemovingCommonLeadingWhitespaceFromLines() -> String {
         var minLeadingWhitespace = Int.max
+
         enumerateLines { line, _ in
-            let lineLeadingWhitespace = line.countOfLeadingCharactersInSet(whitespaceAndNewlineCharacterSet)
-            if lineLeadingWhitespace < minLeadingWhitespace && lineLeadingWhitespace != line.characters.count {
```

Comment:
I don't feel particularly comfortable removing the `lineLeadingWhitespace != line.characters.count` check because I'm not sure what it was guarding against. @jpsim please advise :)